components/eamxx/src/physics/p3/atmosphere_microphysics.cpp

```diff
@@ -275,8 +282,11 @@ void P3Microphysics::initialize_impl (const RunType /* run_type */)
   }
   diag_inputs.ni_activated    = get_field_in("ni_activated").get_view<const Pack**>();
   diag_inputs.inv_qc_relvar   = get_field_in("inv_qc_relvar").get_view<const Pack**>();
+
+//OG why is this not using p3_preproc?
```

Comment:
I think for simplicity, since we can use the field pointer directly for this value there's no need to store a copy in the preproc.